daq/host.py

```diff
@@ -247,6 +248,12 @@ def _prepare(self):
         self._state_transition(_STATE.WAITING, _STATE.INIT)
         self.record_result('sanity', state=MODE.DONE)
         self.record_result('dhcp', state=MODE.EXEC)
+        _ = list(listener(self) for listener in self._dhcp_listeners)
```

Comment:
is the "_ = list" necessary?  Can you just do:

[listener(self) for listening in self._dhcp_listeners]

or does something complain?  (It might about ignored return value)

Reply:
yes. if I don't do assignments, the linter complains